modules/using-a-drift-metric-in-a-credit-card-scenario.adoc

```diff
@@ -0,0 +1,204 @@
++
+[source, bash]
+----
+curl -sk -H "Authorization: Bearer ${TOKEN}" -X POST --location $TRUSTY_ROUTE/info/names \
+  -H "Content-Type: application/json"   \
+  -d "{
+    \"modelId\": \"gaussian-credit-model\",
+    \"inputMapping\":
+      {
+        \"credit_inputs-0\": \"Age\",
+        \"credit_inputs-1\": \"Credit Score\",
+        \"credit_inputs-2\": \"Years of Education\",
+        \"credit_inputs-3\": \"Years of Employment\"
+      },
+    \"outputMapping\": {
+      \"predict-0\": \"Acceptance Probability\"
+    }
+  }"
+----
++ 
+[TIP]
+====
+Define name mappings in TrustyAI to assign memorable names to input or output names. These names can then be used in subsequent requests to the TrustyAI service.
+====
+
+. Verify that TrustyAI has received the data by querying the `/info` endpoint:
++
+[source, bash]
+----
+curl -H "Authorization: Bearer ${TOKEN}" $TRUSTY_ROUTE/info | jq '.["gaussian-credit-model"].data.inputSchema'
+----
+
+. The following output appears as a json file confirming that TrustyAI has successfully received the data:
++
+[source, bash]
+----
+{
+  "items": {
+    "Years of Education": {
+      "type": "DOUBLE",
+      "name": "credit_inputs-2",
+      "columnIndex": 2
+    },
+    "Years of Employment": {
+      "type": "DOUBLE",
+      "name": "credit_inputs-3",
+      "columnIndex": 3
+    },
+    "Age": {
+      "type": "DOUBLE",
+      "name": "credit_inputs-0",
+      "columnIndex": 0
+    },
+    "Credit Score": {
+      "type": "DOUBLE",
+      "name": "credit_inputs-1",
+      "columnIndex": 1
+    }
+  },
+  "nameMapping": {
+    "credit_inputs-0": "Age",
+    "credit_inputs-1": "Credit Score",
+    "credit_inputs-2": "Years of Education",
+    "credit_inputs-3": "Years of Employment"
+  }
+}
+----
+
+. Create a recurring drift monitoring metric using `/metrics/drift/meanshift/request`. This will measure the drift of all recorded inference data against the reference distribution. The body of the payload requires a `modelId` that sets which model to monitor and a referenceTag that determines which data to use as the reference distribution. The values of these fields should match the `modelId` and `referenceTag` inside your data upload payload:
++
+[source, bash]
+----
+curl -k -H "Authorization: Bearer ${TOKEN}" -X POST --location $TRUSTY_ROUTE/metrics/drift/meanshift/request -H "Content-Type: application/json" \
+  -d "{
+        \"modelId\": \"gaussian-credit-model\",
+        \"referenceTag\": \"TRAINING\"
+      }"
+----
+
+. Check the metrics in the OpenShift console under *Observe -> Metrics*:
+.. Set the time window to 5 minutes and the refresh interval to 15 seconds.
+.. In the *Expression* field, enter `trustyai_meanshift`. It may take a few seconds before the cluster monitoring stacks picks up the new metric. You may need to refresh before the new metrics appear, if you're already in the section of the OpenShift console.
```

Comment:
Should this be a note? "It may take a few seconds before the cluster monitoring stacks picks up the new metric. You may need to refresh before the new metrics appear, if you're already in the section of the OpenShift console."